resources/assets/js/app.js

```diff
@@ -0,0 +1 @@
+const autosize = require('autosize');
```

Comment:
Is this actually used anywhere?

Reply:
(I know it's used inline, but I think that's solved by the fact that we're still pulling the script in)